docs/en/ingest-management/elastic-agent/configuration/inputs/input-winlog.asciidoc

```diff
@@ -0,0 +1,372 @@
+
+Example:
+
+[source,yaml]
+----
+name: Application
+ignore_older: 168h
+----
+
+// end::ignore_older-setting
+
+// tag::include_xml-setting
+|
+[id="input-{input-type}-include_xml-setting"]
+`include_xml`
+
+| (boolean) Controls if the raw XML representation of an event is
+included in the data sent by {agent}.
+*{vista_and_newer}*
+
+The XML representation of the event is useful for troubleshooting purposes. The
+data in the fields reported by {agent} can be compared to the data in the XML
+to diagnose problems.
+
+Example:
+
+[source,yaml]
+----
+name: Application
+include_xml: true
+----
+
+*Default:* false
+
+// end::include_xml-setting
+
+
+include::input-shared-settings.asciidoc[tag=index-setting]
+
+include::input-shared-settings.asciidoc[tag=keep_null-setting]
+
+// tag::level-setting
+|
+[id="input-{input-type}-level-setting"]
+`level`
+
+| (list) A list of event levels to include. The value is a comma-separated list of
+levels. *{vista_and_newer}*
+
+[horizontal]
+.Levels and Values
+critical, crit:: 1
+error, err:: 2
+warning, warn:: 3
+information, info:: 0 or 4
+verbose:: 5
+
+Example:
+
+[source,yaml]
+----
+name: Security
+level: critical, error, warning
+----
+
+// end::level-setting
+
+// tag::no_more_events-setting
+|
+[id="input-{input-type}-no_more_events-setting"]
+`no_more_events`
+
+| (string) The action that the event log reader should take when it
+receives a signal from Windows that there are no more events to
+read. It can either `wait` for more events to be written (the default
+behavior) or it can `stop`. The overall {agent} process will stop when
+all of the individual event log readers have
+stopped. *{vista_and_newer}*
+
+Setting `no_more_events` to `stop` is useful when reading from archived event
+log files where you want to read the whole file then exit.
+
+Example:
+
+[source,yaml]
+----
+name: Application
+no_more_events: stop
+----
+
+*Default:* wait
+
+// end::no_more_events-setting
+
+include::input-shared-settings.asciidoc[tag=processors-setting]
+
+// tag::provider-setting
+|
+[id="input-{input-type}-provider-setting"]
+`provider`
+
+| (list) A list of providers (source names) to include. The value is a YAML list.
+*{vista_and_newer}*
+
+Example:
+
+[source,yaml]
+----
+name: Application
+provider:
+  - Application Error
+  - Application Hang
+  - Windows Error Reporting
+  - EMET
+----
+
+You can obtain a list of providers associated with a log by using PowerShell.
+Here is an example showing the providers associated with the Security log.
+
+[source,sh]
+----
+PS C:\> (Get-WinEvent -ListLog Security).ProviderNames
+DS
+LSA
+SC Manager
+Security
+Security Account Manager
+ServiceModel 4.0.0.0
+Spooler
+TCP/IP
+VSSAudit
+Microsoft-Windows-Security-Auditing
+Microsoft-Windows-Eventlog
+----
+
+// end::provider-setting
+
+include::input-shared-settings.asciidoc[tag=tags-setting]
+
+
+|===
+
+[[input-winlog-experimental-settings]]
+=== Experimental settings
```

Comment:
For how long are these experimental? Time to move them to Beta/GA?

Reply:
that I don't know.  @andrewkroh do you know?

Reply:
I double checked with @andrewkroh and we can move them to production.  I'll open a PR to do the same in the winlogbeat docs.